package_test.go

```diff
@@ -657,20 +657,20 @@ func (s *PackageSuite) TestValidGetAll(c *C) {
 		inputs:   []any{Address{ID: 1000}, Person{ID: 30}},
 		slices:   []any{&[]*Manager{}, &[]*Person{}, &[]*Address{}},
 		expected: []any{&[]*Manager{{30, "Fred", 1000}}, &[]*Person{{30, "Fred", 1000}}, &[]*Address{{1000, "Happy Land", "Main Street"}}},
-	}, {
-		summary:  "nothing returned",
-		query:    "SELECT &Person.* FROM person WHERE id = $Person.id",
-		types:    []any{Person{}},
-		inputs:   []any{Person{ID: 1243321}},
-		slices:   []any{&[]*Person{}},
-		expected: []any{&[]*Person{}},
 	}, {
 		summary:  "select into maps",
 		query:    "SELECT &M.name, &CustomMap.id FROM person WHERE name = 'Mark'",
 		types:    []any{sqlair.M{}, CustomMap{}},
 		inputs:   []any{},
 		slices:   []any{&[]sqlair.M{}, &[]CustomMap{}},
 		expected: []any{&[]sqlair.M{{"name": "Mark"}}, &[]CustomMap{{"id": int64(20)}}},
+	}, {
+		summary:  "run insert with GetAll",
```

Comment:
I think we should name the test something like `GetAll returns no error when there are no outputs`, that way we won't be tempted to refactor it if we add tests for inserts later.